Dockerfile

```diff
@@ -40,6 +40,10 @@ RUN mv /etc/ssl/certs/ca-bundle.crt /etc/ssl/certs/ca-bundle.crt.org && \
     mkdir -p /etc/pki/ca-trust/extracted/openssl
 
 USER root
+# this is a temporary solution for an update to the crds_s3_get plugin
```

Comment:
I'd guess this works and is temporary,  but unless crds_s3_get is needed before caldp is installed,  crds_s3_get will be installed in the conda environment's bin directory so you shouldn't have to do this.